pyenvbuilder/utils.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import subprocess
+from pathlib import Path
+
+
+logger = logging.getLogger(__name__)
+
+
+def validate_installed(args):
+    '''
+    Invokes the args and returns True if successful
+    '''
+    try:
+        if subprocess.run(args, check=True, stdout=subprocess.DEVNULL):
+            return True, f''
+    except subprocess.CalledProcessError as err:
+        logger.error(err)
+        return False, err
+    except FileNotFoundError:
+        msg = f'{args} not found, please install {args} before proceeding.'
+        logger.error(msg)
+        return False, msg
+
+
+def validate_path(args):
+    '''
+    Validates if args is valid path
+    Returns True/False and returns if is_file/is_dir
+    '''
+    f_path = Path(args)
+    ret_arg = {'is_file': False, 'is_dir': False}
+
+    if f_path.exists():
+        if f_path.is_file():
+            ret_arg['is_file'] = True
+        elif f_path.is_dir():
+            ret_arg['is_dir'] = True
+        return True, ret_arg
+    else:
+        return False, ret_arg
+
+
+def locate_files(**kwargs):
```

Comment:
Why `**kwargs`?

Reply:
I switched this to accept `files` instead of the `**kwargs`...
So now, when calling this function from `check` or `create` I am passing in `kwargs.get('files')` - before I was passing `**kwargs` - please let me know if this is fine